CMakeLists.txt

```diff
@@ -580,6 +580,8 @@ if(NOT ${use_installed_dependencies})
     endif()
 endif()
 
+include(dependencies.cmake)
```

Comment:
Not this time but the next, we need to consolidate the umock-c either put all of it in dependencies.cmake or have it here.